nengo_gui/guibackend.py

```diff
@@ -295,7 +295,7 @@ def get_session(self):
         return session
 
     def log_message(self, format, *args):
-        logger.info(format, *args)
+        logger.debug(format, *args)
```

Comment:
You're right, for some reason those messages did not appear on my Macbook (potentially I was running the wrong version of Nengo GUI). I agree that the `debug` log level makes more sense.